api/dataprocessor_test.go

```diff
@@ -611,7 +611,10 @@ func TestAlignRequests(t *testing.T) {
 				reqRaw("a", 0, 30, 800, 10, consolidation.Avg),
 				reqRaw("b", 0, 30, 800, 60, consolidation.Avg),
 			},
-			[]mdata.AggSetting{},
+			mdata.AggSettings{
+				35 * 24 * 60 * 60,
```

Comment:
The reason I changed the tests is because `alignRequests` now expects the second parameter to be of type `AggSettings{}` and that type requires a number to be there as the default TTL. You're right that in the case of these tests that number does not matter at all, so I've just put the default value there.